paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetConfigurationTest.kt

```diff
@@ -81,7 +81,8 @@ class CustomerSheetConfigurationTest {
             "defaultBillingDetails",
             "headerTextForSelectionScreen",
             "preferredNetworks",
-            "build"
+            "build",
+            "allowsRemovalOfLastSavedPaymentMethod",
```

Comment:
Actually, should this test against `CustomerSheet.Configuration`? That way, we could make sure that we don’t forget to add the builder method. That has happened in PaymentSheet in the past.

Reply:
A weird test in my book. I don't see it adding much value myself.